data/blog/funnel-101.mdx

```diff
@@ -0,0 +1,264 @@
+---
+title: 'Funnel 101: sharing your local developer preview with the world'
+date: 2023-03-10
+authors: ['xe']
+tags:
+  - funnel
+summary: 'Tailscale Funnel lets you share a local service with the internet. Today Xe covers how you can use this productively by sharing the development copy of an application to the world.'
+---
+
+Tailscale lets you share things with your coworkers, your family, and your
+friends; but not everyone else. This is great when you're testing things with
+your coworkers or playing Minecraft with your friends, but sometimes you need
+to test things against the real world. Tailscale Funnel allows you to expose
+your local services to the public Internet.
+
+Today I'm going to walk you through the basics of funnel and show you how to
+use it with a live demo.
+
+In order to follow along, you will need the following:
+
+- An invite to the Funnel feature preview (ask us [on
+  Mastodon](https://hachyderm.io/@tailscale)!)
+- A development machine running Linux, macOS, or Windows
+  - Be sure [funnel is enabled in the tailnet policy file
+    settings](https://tailscale.com/kb/1223/tailscale-funnel/) for that machine
```

Comment:
the nit/personal pref here is I try to avoid single-item lists, even as sublists, and would personally just make this part of parent list item

the possible bug report is that nested lists look like they've got some sort of weird spacing issue in the preview